crates/introduction_to_fire_dynamics/src/chapter_6/equation_6_33.rs

```diff
@@ -0,0 +1,36 @@
+pub fn time_to_ignition_thermally_thin(
```

Comment:
```suggestion
pub fn time_to_ignition(
```

Same here. I don;t think anyone is going to use this equation because of the name. They will use it because they know it is equation 6.33.

Reply:
Addressed.